hack/verify-flags/known-flags.txt

```diff
@@ -643,6 +643,7 @@ static-pods-config
 stats-port
 stop-services
 storage-media-type
+storage-backend
```

Comment:
This line is added to pass test hack/verify-flags-underscore.py
I have asked a question about it here:
https://github.com/kubernetes/kubernetes/pull/30738#pullrequestreview-26733630